analyses/MI_cd_2010/doc_MI_cd_2010.md

```diff
@@ -0,0 +1,24 @@
+# 2010 Michigan Congressional Districts
+
+## Redistricting requirements
+in Michigan, according to (http://www.legislature.mi.gov/(S(xxvumgge0jwzkeswmwt0bh4v))/mileg.aspx?page=GetObject&objectname=mcl-Article-IV-6), districts must:
```

Comment:
This should be `[Article IV, Section 6](http://www.legislature.mi.gov/(S(xxvumgge0jwzkeswmwt0bh4v))/mileg.aspx?page=GetObject&objectname=mcl-Article-IV-6) of the state constitution, districts must:`

The square brackets are what the text will be and the parens are the link